### PR TITLE
BCDA-4239 - Add method to generate ALR jobs in service.go

### PR DESCRIPTION
**Pull request by mtrang1263**
### Fixes [BCDA-4239](https://jira.cms.gov/browse/BCDA-4239)

Adding method to service package to generate ALR jobs.

In this PR, we are piloting the usage of [testfixtures](https://github.com/go-testfixtures/testfixtures). These fixtures allow us to load data into our test postgres database via fixtures file. This will allow our test code to focus on validation instead of dealing with the loading of data.

### Change Details
1. Added alr.go to the service package that is responsible for creating ALR jobs.
2. Added new utilities to create test databases while ensuring consistent schemas. This will allow us to leverage the testfixtures approach while still allowing for parallel tests.

### Security Implications
- [x] new software dependencies
We are using two new test scoped dependencies.

1. https://github.com/go-testfixtures/testfixtures - Used for loading the database via files
2. https://github.com/golang-migrate/migrate - Used to ensure consistent schema in the test database.

- [ ] security controls or supporting software altered
- [ ] new data stored or transmitted
- [ ] security checklist is completed for this change
- [ ] requires more information or team discussion to evaluate security implications
- [x] no PHI/PII is affected by this change

### Acceptance Validation
CI Passes. This functionality is not exposed to the caller. We have follow up tickets to achieve that.
### Feedback Requested
Review testing strategy to see if it will meet the needs of our improved testing effort.

### CODE REVIEW

**Review by CosmicSyntax — DISMISSED** [review]
LGTM

**Review by SMLuthi — DISMISSED** [review]
LGTM! The new fixture work is really slick.

File: bcda/database/connection_test.go [hunk 1]
```diff
@@ -40,9 +40,9 @@ func TestHealthCheck(t *testing.T) {
 	ctx, cancel := context.WithCancel(context.Background())
 	startHealthCheck(ctx, Connection, QueueConnection, 100*time.Microsecond)
 	// Let some time elapse to ensure we've successfully ran health checks
-	time.Sleep(10 * time.Millisecond)
+	time.Sleep(50 * time.Millisecond)
```
Comment (mtrang1263) [inline bcda/database/connection_test.go:43]: Ran this locally and realized that the 10ms window was a bit too tight

**Review by CosmicSyntax — APPROVED** [review]
LGTM

**Pull request merged**